### PR TITLE
Bringing the Supporter Program to Phase 2

### PR DESCRIPTION
**Pull request by SaraJo**

### CODE REVIEW

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Summary
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js, gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+### Phase 1 Components
+
+**Launch Announcement
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF
```
Comment (tobie) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:12]: Curious why the LF is involved, here, or is this just OpenJSF staff?

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### Phase 1 Components
+
+**Launch Announcement
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF 
+We will determine the best date to post with the content team
+
+Design requrements:
```
Comment (tobie) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:15]:
```suggestion
Design requirements:
```

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* Image of a country with multiple JS Libraries
```
Comment (tobie) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:16]: This seems obvious, but we want to make sure this country doesn't look like a country in particular.

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+**Website Changes
+
+A signup page where supporters can become members and pay for their yearly membership. We plan to adopt the auto join functionality from the Linux Foundation. 
+
+An automated workflow to add new supporters to monthly email
+* Existing subscribers will not be removed
+* They can request removal at any time
+* Zapier or IFTT can work if no other solution
+
+A new platform for the weekly emails, and a designed template.
```
Comment (tobie) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:28]: Is @jorydotcom going to continue being the person in charge of this newsletter?

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* Binocs
+* Fanny pack
+* Tourguide flags
+* Map
+* Passport person is traveling to see everything, doesn’t have a destination
+* Topography instead of countries
+* Play on the passport is the digital badge
+
+
+Logo Things to avoid:
```
Comment (tobie) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:48]: Great to have this section, here. Thank you!

**Review by tobie — APPROVED** [review]
This LGTM. A few questions out of curiosity, bit nothing that's intended to be a blocker.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by kborchers — APPROVED** [review]

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 1 above)
Comment (MylesBorins) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:12]: The OpenJS staff are actually LF staff contracted to us... so this is primarily semantics

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 3 above)
Comment (MylesBorins) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:16]: I think a big square could be a way to ensure that it doesn't resemble any other nation (thinking the big yellow JS square)

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF 
+We will determine the best date to post with the content team
+
+Design requrements:
+* Image of a country with multiple JS Libraries
+* Image will be created by LF internal team, we also can source iterations of it on Twitter or Github
+
+**Website Changes
```
Comment (MylesBorins) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:19]:
```suggestion
**Website Changes**
```
Is this supposed to be bold?

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# JavaScriptLandia - Phase 1 - Spec
+## October 2020 
+
+### Summary
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js, gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+### Phase 1 Components
+
+**Launch Announcement
```
Comment (MylesBorins) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:9]:
```suggestion
**Launch Announcement**
```

is this supposed to be bold?

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+* They can request removal at any time
+* Zapier or IFTT can work if no other solution
+
+A new platform for the weekly emails, and a designed template. 
+    
+A page to reflect current supporters on the website
+
+A swag store for supporters (turnkey)
+
+**A Logo and Badge
```
Comment (MylesBorins) [inline proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:34]:
```suggestion
**A Logo and Badge**
```

is this supposed to be bold?

**Review by MylesBorins — APPROVED** [review]
LGTM

Left some tiny nits inline that don't block this moving forward

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 1 above)
Comment (tobie) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:12]: Well, whether this effort is an OpenJSF effort or a LF/OpenJSF effort does have different implications. It wasn’t clear to me from reading this which one it was. I think it’s worth clarifying.

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 5 above)
Comment (SaraJo) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:48]: Agreed. Thanks.

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 3 above)
Comment (SaraJo) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:16]: Agreed.

File: proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md (hunk 4 above)
Comment (SaraJo) [reply proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md:28]: Yes, definitely.

**Pull request merged**